app/extensions.js

```diff
@@ -467,6 +519,16 @@ module.exports.init = () => {
     extensionActions.extensionDisabled(config.torrentExtensionId)
   }
 
+  // ipfsExtension
+  console.log('IPFS: is extension enabled', getSetting(settings.IPFS_ENABLED))
+  if (getSetting(settings.IPFS_ENABLED)) {
+    extensionInfo.setState(config.ipfsExtensionId, extensionStates.REGISTERED)
+    loadExtension(config.ipfsExtensionId, getExtensionsPath('ipfs'), generateIPFSManifest(), 'component')
+  } else {
```

Comment:
I can confirm that `getSetting(settings.IPFS_ENABLED))` prints true, which means that the extension should be loaded.